pkg/kubelet/kubelet.go

```diff
@@ -525,7 +525,6 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		experimentalHostUserNamespaceDefaulting: utilfeature.DefaultFeatureGate.Enabled(features.ExperimentalHostUserNamespaceDefaultingGate),
 		keepTerminatedPodVolumes:                keepTerminatedPodVolumes,
 		nodeStatusMaxImages:                     nodeStatusMaxImages,
-		lastContainerStartedTime:                newTimeCache(),
```

Comment:
looks like a bad rebase dropped this

Reply:
Yes, there was some problem with that, thought that recovered everything, but haven't noticed it.